src/main/java/htsjdk/tribble/readers/AsciiLineReader.java

```diff
@@ -93,6 +105,10 @@ public final String readLine(final PositionalBufferedStream stream) throws IOExc
             if (c == LINEFEED || c == CARRIAGE_RETURN) {
                 if (c == CARRIAGE_RETURN && stream.peek() == LINEFEED) {
                     stream.read(); // <= skip the trailing \n in case of \r\n termination
+                    this.lineTerminatorLength = 2;
+                }
+                else {
```

Comment:
How meta ;)  In all seriousness though I thought putting the else onto the next line was user-choice in htsjdk & picard.  Certainly the rule used to be the opposite (always put it on a new line).